guides/common/modules/proc_assigning-a-host-to-a-specific-location.adoc

```diff
@@ -2,7 +2,13 @@
 = Assigning a host to a specific location
 
 Use this procedure to assign a host to a specific location.
-For general information about locations and how to configure them, see {ContentManagementDocURL}Creating_a_Location_content-management[Creating a Location] in _{ContentManagementDocTitle}_.
+For general information about locations and how to configure them,
+ifdef::satellite[]
+see {AdministeringDocURL}Managing_Locations_admin[Managing locations] in _{AdministeringDocTitle}_.
+endif::[]
+ifndef::satellite[]
+see {ManagingOrganizationsLocationsDocURL}Managing_Locations_managing-organizations-locations[Managing locations] in _{ManagingOrganizationsLocationsDocTitle}_.
```

Comment:
Just a suggestion, a more future-proof alternative:
```suggestion
see {ManagingOrganizationsLocationsDocURL}[_{ManagingOrganizationsLocationsDocTitle}_].
```
It's a very small guide so users shouldn't have any trouble locating the chapter on locations. And when pointing to a guide, there is little risk in the link getting broken if the chapter ID changes in the future.

Reply:
Good call -> applied.